rslib/src/stats/graphs/reviews.rs

```diff
@@ -38,7 +38,7 @@ impl GraphsContext {
                     count.filtered += 1;
                     time.filtered += review.taken_millis;
                 }
-                RevlogReviewKind::Manual => unreachable!(),
+                RevlogReviewKind::Manual | RevlogReviewKind::Rescheduled => unreachable!(),
```

Comment:
You forgot to exclude the rescheduled case at the top, so it's reachable.

Reply:
aea26e9807f710e3f22aaca9c279d6e4540974ff